spack/wdmapp/packages/wdmapp/package.py

```diff
@@ -53,7 +53,7 @@ class Wdmapp(BundlePackage):
     depends_on('tau@develop +adios2 ~libunwind ~pdt +mpi', when='+tau')
 
     # variant +effis
-    depends_on('effis python-type=minimal', when='+effis')
+    depends_on('effis -python -compose', when='+effis')
```

Comment:
@suchyta1 IIUC, this spec produce an effis install that does *not* provide `effis-compose.py` and `effis-submit` scripts and a subsequent `spack install effis python-type=full` command will install the scripts.  Is that correct?

Reply:
@cwsmith `-compose` won't produce a working `effis-comose.py`. (`effis-submit` is just a shell script, but is pretty useless without the compose.)

You'll want `+compose +python` for the front end, which I made the default. I got rid of `python-type` in favor of `compose` and `python`, which are just true or false (for the reasons above.)

Reply:
This also made me realize, I forgot to add a `conflicts` to make sure `+compose` uses `+python`.